docs/Logging.md

```diff
@@ -11,13 +12,35 @@ this purpose.
 
 As Fastify is focused on performance, it uses [pino](https://github.com/pinojs/pino) as its logger, with the default log level, when enabled, set to `'info'`.
 
-Enabling the logger is extremely easy:
+Enabling the production JSON logger:
 
 ```js
 const fastify = require('fastify')({
   logger: true
 })
+```
+
+Enabling the logger with sensible configuration for both local development and production environment requires bit more configuration:
+```js
+const { NODE_ENV} = process.env
```

Comment:
```suggestion
const { NODE_ENV } = process.env
```

But really, I'd rather not even mention `process.env.NODE_ENV`. It is a terrible pattern to perpetuate. `NODE_ENV` assumes _development_ first instead of _production_ first.

Reply:
It's really down to the application and how it is implemented. In the example as written, there isn't any room other than inspecting an environment variable. But that variable doesn't have to be `NODE_ENV`. It could be `process.env.ENABLE_DEV = true` or pretty much anything else.